catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SeedCommand.java

```diff
@@ -91,50 +83,21 @@
         EXECUTOR.allowCoreThreadTimeOut(true);
     }
 
+    private final Predicate<Metacard> isNonCachedResource = metacard -> {
+        Attribute cached = metacard.getAttribute(RESOURCE_CACHE_STATUS);
+        return cached == null || !((Boolean) cached.getValue());
```

Comment:
I would like to see an instanceof check to make sure getValue() can be cast to a Boolean.